app/views/foreman_ansible/job_templates/ansible_windows_updates.erb

```diff
@@ -0,0 +1,187 @@
+    Center Configuration Manager (SCCM), or similar corporate update server environments
+    may default to those managed update sources instead of the Windows Update catalog.\r\nmanaged_server
+    Use a managed server catalog. For environments utilizing Windows Server Update
+    Services (WSUS), Systems Center Configuration Manager (SCCM), or similar corporate
+    update servers, this option selects the defined corporate update source.\r\nwindows_update
+    Use the Microsoft Windows Update catalog.\r\n\r\nDefault is 'default'"
+  options: "default\r\nmanaged_server\r\nwindows_update"
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+- name: state
+  required: false
+  input_type: user
+  description: "Controls whether found updates are downloaded or installed or listed\r\nThis
+    module also supports Ansible check mode, which has the same effect as setting
+    state=searched\r\n\r\nDefault is 'searched'"
+  options: "installed\r\nsearched\r\ndownloaded"
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+- name: use_scheduled_task
+  required: false
+  input_type: user
+  description: "Will not auto elevate the remote process with become and use a scheduled
+    task instead.\r\nSet this to true when using this module with async on Server
+    2008, 2008 R2, or Windows 7, or on Server 2008 that is not authenticated with
+    basic or credssp.\r\nCan also be set to true on newer hosts where become does
+    not work due to further privilege restrictions from the OS defaults."
+  options: "false\r\ntrue"
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+- name: whitelist
+  required: false
+  input_type: user
+  description: "A list of update titles or KB numbers that can be used to specify
+    which updates are to be searched or installed.\r\nIf an available update does
+    not match one of the entries, then it is skipped and not installed.\r\nEach entry
+    can either be the KB article or Update title as a regex according to the PowerShell
+    regex rules.\r\nThe whitelist is only validated on updates that were found based
+    on category_names. It will not force the module to install an update if it was
+    not in the category specified."
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+model: JobTemplate
+job_category: Ansible Playbook
+provider_type: Ansible
+kind: job_template
+organizations:
+- My_Organization
+locations:
+- My_Location
+%>
+
+- hosts: all
+  vars:
+    updates: []
+    <% if input('state').blank? %>
+    state: searched
+    <% else %>
+    state: <%= input('state') %>
+    <% end %>
+
+  tasks:
+    - name: "{{ state | replace('ed','') | capitalize }} Windows Updates"
+      win_updates:
+        <% unless input('blacklist').blank? %>
+        blacklist: <%= input('blacklist') %>
```

Comment:
```suggestion
        blacklist: <%= input('blocklist') %>
```

Reply:
In the ansible module it´s also called blacklist, so I think it would be good to leave it the same as in the ansible module

Reply:
ist im commit enthalten